src/database/models/bookedaccommodations.js

```diff
@@ -0,0 +1,26 @@
+module.exports = (sequelize, DataTypes) => {
+  const bookedaccommodations = sequelize.define('bookedaccommodations', {
```

Comment:
I think what we're trying to model here is a booking or reservation. I suggest you name this model 'Booking' or `Reservation`